ksql-rest-app/src/test/java/io/confluent/ksql/api/ApiTest.java

```diff
@@ -54,9 +52,12 @@
 
 public class ApiTest extends BaseApiTest {
 
-  private static final Logger log = LoggerFactory.getLogger(ApiTest.class);
+  protected static final Logger log = LoggerFactory.getLogger(ApiTest.class);
+
+  protected static final List<JsonObject> DEFAULT_INSERT_ROWS = generateInsertRows();
 
   @Test
+  @CoreApiTest
```

Comment:
What's the thinking behind the new annotation?

Reply:
I didn't think we needed to run all the tests in `ApiTest` with basic auth (some of the negative tests also won't work since the expected error code changes with auth). Between duplicating code for the tests I did want to run with basic auth and adding an annotation to filter tests on, I went with the annotation approach. This way it's re-usable in that if we add other auth mechanisms, or other non-auth client/server mechanisms we want to run ApiTests against, the annotation allows for these future use cases as well.